src/user/models/user/user.entity.ts

```diff
@@ -96,9 +95,6 @@ export class User extends IEntity {
   @Column({ type: 'float', default: 0 })
   stakingBalance: number;
 
-  @Column({ length: 256, default: Blockchain.DEFICHAIN })
-  blockchain: Blockchain;
-
```

Comment:
Migration update is missing

Reply:
good point, but we don't need to create a new migration file. we just need to adapt an existing one, as this column never got created on dev environment